demo/full/styles/style.css

```diff
@@ -409,3 +573,107 @@ body {
 .time-indicator {
   height: 100%;
 }
+
+.nav-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 15px 0 15px;
+  background-color: #222424;
+  color: #ffffff;
+}
+
+.nav-header a {
+  color: #ffffff;
+}
+
+.header-links-buttons {
+  display: flex;
+  align-items: center;
+}
+
+.header-links-buttons > * {
+  margin: 3px;
+}
+
+.github-button {
+  margin: 5px;
+}
+
+.drm-settings {
+  background: #ffffff;
+  padding: 14px 14px 2px 14px;
+  margin: 12px 0 12px 0;
+  border: solid 1px #a9a9a9;
+}
+
+.drm-settings .input {
+  border: solid 1px #a9a9a9;
+}
+
+.white-select select {
+  /* styling */
+  background-color: white;
+  display: inline-block;
+  font: inherit;
+  height: 41px;
+  padding: 10px 25px 10px 15px;
+
+  /* reset */
+
+  margin: 0;
+  -webkit-box-sizing: border-box;
+  -moz-box-sizing: border-box;
+  box-sizing: border-box;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+
+  background-image: url("../assets/select_arrow_black.svg");
+  background-size: 8px;
+  background-repeat: no-repeat;
+  background-position: calc(100% - 8px) 50%;
+
+  border: 1px solid #a9a9a9;
+}
+
+.white-select option {
+  padding: 8px;
+}
+
+.black-knob select {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Expected selector ".black-knob select" to come before selector ".knob-value > select:focus" ([css:S4664](https://sonarcloud.io/organizations/rx-player/rules#rule_key=css%3AS4664))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=312&issues=AWXYTOxW_eEZPtUmDdcb&open=AWXYTOxW_eEZPtUmDdcb)

[ISSUE_KEY]: AWXYTOxW_eEZPtUmDdcb